LibTest/typed_data/Float32x4List/setAll_A04_t01.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion void setAll(int index, Iterable<E> iterable)
+/// ...
+/// If iterable is based on this list, its values may change /during/ the setAll
+/// operation.
+///
+/// @description Checks that if iterable is based on this list, its values may
+/// change during the `setAll` operation.
+/// @author sgrekhov22@gmail.com
+
+import "dart:typed_data";
+import "../../../Utils/expect.dart";
+
+Float32x4 f32x4(v) => new Float32x4.splat(v);
+
+listEquals(List<Float32x4> expected, Float32x4List actual) {
+  Expect.equals(expected.length, actual.length);
```

Comment:
It may make sense to add this length check to other `listEquals` variants. For example the Int32x4List version of this test doesn't check lengths and the test can pass if `expected` is larger.

Reply:
I believe this has been resolved.